trader/Task task
---
bool auctionReady
